system_tests/datastore.py

```diff
@@ -25,8 +25,18 @@
 from system_tests import populate_datastore
 
 
-client.DATASET = TESTS_DATASET
-CLIENT = datastore.Client()
+class Config(object):
+    """Run-time configuration to be modified at set-up.
+
+    This is a mutable stand-in to allow test set-up to modify
+    global state.
+    """
+    CLIENT = None
+
+
+def setUpModule():
+    client.DATASET = TESTS_DATASET
+    Config.CLIENT = datastore.Client()
```

Comment:
Why `Config.CLIENT` here and `self.client` in `bigquery.py`?  Shouldn't we follow a consistent pattern?